### PR TITLE
Nwd nnw Lukasz K/Nanautzin

### PR DESCRIPTION
**Pull request by lukasz-kukulka**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @lukasz-kukulka

**Comment by ziobron** [comment]
⏱ 3 XP extra for @lukasz-kukulka for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,24 @@
 #pragma once
+#include <cstdlib>
+#include <cmath>
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    int temp, left = lhs, right = rhs;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:6]: One definition/ declaration per line.

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+    if (left < right){
+        right = lhs;
+        left = rhs;
+    }
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:10]: if (lhs > rhs) {
   std::swap(lhs, rhs)
}

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+    return abs( left );
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if(lhs == 0 || rhs == 0){
+        return 0;
+    } else
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:22]: else is redundant. Also you should add brackets

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**